### ISSUE TITLE
Issue Starting Program

### ISSUE THREAD

**Issue by unconverted** [mscrivo/ootd#30]
I was using the program for a while and decided to upgrade to the newest version. This gave me an error on startup. Downgraded back to an older version (I believe it was 2.5 that worked), and that was good for a while. Now it starts to hang up regularly and cannot add or edit calendar items. So I went to upgrade again as I see there was a really new version (3.3.0) and I still get an error preventing it from starting.

Error is:
There was an error initializing the application: Exception occurred. (Exception from HRESULT: 0x80020009 (DISP_E_EXCEPTION).

Microsoft Outlook 2013 (15.0.4771.1000) MSO (15.0.4771.1001) 32-bit
Windows 7 Professional - 64bit os

I tried to run the program as Admin and no dice.

Any thoughts or things I could try?

Thank-you.

**Comment by mscrivo** [comment]
Indicates an issue with your Office configuration.  Can you try
repairing/reinstalling office?

**Comment by unconverted** [comment]
Thank-you for the reply! I tried repairing office and that didn't help. I then did a full uninstall of OOTD and manually deleted the program folder then installed the newest OOTD. That worked, but it regularly hangs up. I can adjust the brightness of the ootd window but nothing else.

If i click on some of the buttons a few times it gives me 'Unhandled exception has occured' 'RPC server is unavailable. (Exception from HRESULT:0x800706BA)
